Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3500,7 +3500,7 @@ MaskWebGLStringsEnabled:
 
 MasonryEnabled:
   type: bool
-  status: testable
+  status: preview
```

Comment:
I wouldn't say masonry is ready, this basic demo doesn't work on trunk: https://yisibl.github.io/houdini-demo/masonry/index-native.html